pkg/devfile/generator/generators.go

```diff
@@ -158,6 +159,7 @@ type DeploymentParams struct {
 	Containers        []corev1.Container
 	Volumes           []corev1.Volume
 	PodSelectorLabels map[string]string
+	Replicas          int32
```

Comment:
I would set the Replicas as a pointer, to make it optional (the nil value indicating the user does not want to set it). The 0 value on which you test below is a valid value a user would want to set.

Reply:
Makes sense. Does https://github.com/devfile/library/pull/129/commits/145564efc02c37f573608a4875e81009f3da3427 look good?